src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeSquare;
+    private int track;
+
+    public Bulldozer(int bladeSquare, int track, String name, String color) {
+        this.bladeSquare = bladeSquare;
+        this.track = track;
+        super.setName(name);
+        super.setColor(color);
```

Comment:
Create constructor in Machina class and here add super(name, color) from the start.